app/controllers/rating_votes_controller.rb

```diff
@@ -12,9 +12,9 @@ def create
     rating_vote.rating = rating_vote_params[:rating].to_f
     rating_vote.group = rating_vote_params[:group]
     if rating_vote.save
-      redirect_back(fallback_location: "/mod")
+      render json: { result: "Success" }
     else
-      render json: { result: "Not Upserted Successfully" }
+      render json: { error: rating_vote.errors.full_messages.to_sentence }, status: :unprocessable_entity
```

Comment:
Hmm I think this will affect the functionality in `/:article_path/mod`. I think you'll either need to change the render or change the functionality a bit for the experience level rating in the article `/mod` page.

Reply:
Good point; I think I'll need team input on this @jacobherrington